app/services/usps_in_person_proofing/enrollment_helper.rb

```diff
@@ -0,0 +1,85 @@
+module UspsInPersonProofing
+  class EnrollmentHelper
+    def save_in_person_enrollment(user, profile, pii)
+      enrollment = InPersonEnrollment.create!(
+        profile: profile,
+        user: user,
+        current_address_matches_id: pii[:same_address_as_id],
+        selected_location_details: selected_location_details,
+      )
+
+      enrollment_code = create_usps_enrollment(enrollment, pii)
+      return unless enrollment_code
+
+      # update the enrollment to status pending
+      enrollment.enrollment_code = enrollment_code
+      enrollment.status = :pending
+      enrollment.save!
+
+      send_ready_to_verify_email(user, pii, enrollment)
+    end
+
+    def send_ready_to_verify_email(user, pii, enrollment)
+      user.confirmed_email_addresses.each do |email_address|
+        UserMailer.in_person_ready_to_verify(
+          user,
+          email_address,
+          first_name: pii['first_name'],
+          enrollment: enrollment,
+        ).deliver_now_or_later
+      end
+    end
+
+    def usps_proofer
+      if IdentityConfig.store.usps_mock_fallback
+        UspsInPersonProofing::Mock::Proofer.new
+      else
+        UspsInPersonProofing::Proofer.new
+      end
+    end
+
+    def create_usps_enrollment(enrollment, pii)
+      applicant = UspsInPersonProofing::Applicant.new(
+        {
+          unique_id: enrollment.usps_unique_id,
+          first_name: pii['first_name'],
+          last_name: pii['last_name'],
+          address: pii['address1'],
+          # do we need address2?
+          city: pii['city'],
+          state: pii['state'],
+          zip_code: pii['zipcode'],
+          email: 'no-reply@login.gov',
+        },
+      )
+      proofer = usps_proofer
+
+      response = proofer.request_enroll(applicant)
+      response['enrollmentCode']
+    end
+
+    def selected_location_details
```

Comment:
I think we had some timing issues between the merge of this pull request and #6624. It appears that this regressed to using the hard-coded value (and is also in the old format).

Impact is that the location will always be shown as Baltimore, and hours aren't appearing correctly:

![image](https://user-images.githubusercontent.com/1779930/181517235-6e8f7f94-963a-4ad0-a7da-7f43d79b4b46.png)

Reply:
And it may be tricky to fix too, since we can't really rely on the location to live in the session for GPO confirmation 😬